content/docs/1.4/scalers/cron.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.0+"
+maintainer = "Community"
+description = "Scale applications based on Cron Schedules."
+go_file = "cron_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `cron` trigger that scales based on a Cron Schedule.
+
+```yaml
+triggers:
+- type: cron
+  metadata:
+    # Required
+    timezone: <time zone the cron should be run>
+    start: <start of cron>
+    end: <end of cron>
+    metricName: replicacount
+    desiredReplicas: "10"
+```
+
+The `start` indicates the start of the Cron schedule and end indicates the end of the Cron schedule.
```

Comment:
Would be good to provide a bit more documentation on the different parameters similar to https://keda.sh/docs/1.4/scalers/azure-monitor/

Reply:
sure